ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -2143,6 +2143,9 @@ DX3SC:
   - leaving_cooling_coil_temperature_sensor
   - high_supply_air_temperature_alarm
   - low_supply_air_temperature_alarm
+  - compressor_current_sensor_1
```

Comment:
How can you have 3 current sensors but only one speed command? Please confirm if this is actually what the device looks like.